flexget/plugins/input/anidb_feed.py

```diff
@@ -0,0 +1,479 @@
+                                    dt_node.tagName = NAMESPACE_NAME + ':%s' % dt_node.childNodes[0].nodeValue.replace(' ', '_')
+                                    dt_node.setAttribute('name', dt_node.childNodes[0].nodeValue)
+                                    dt_node.setAttribute('value', dd_nodes[idx].childNodes[0].nodeValue)
+                                    dt_node.removeChild(dt_node.childNodes[0])
+                                    node_ns.removeChild(dd_nodes[idx])
+                                    entry.appendChild(dt_node)
+        except Exception as ex:
+            logging.debug('Unable to rename nodes in XML: %s' % ex)
+            return None
+        return dom.toxml()
+
+    def set_ns_attribute(self, name, xml_entry, entry, in_ns_name, out_prefix_name=None, in_type=basestring):
+        tagname = in_ns_name + '_' + name  # feedparser represents those as nsname_tagname
+        value = None
+        if tagname in entry and entry.get(tagname) is not None:
+            value = entry.get(tagname)  # use existing, but do type check
+        elif tagname in xml_entry:
+            if isinstance(xml_entry[tagname], dict) and 'value' in xml_entry[tagname]:
+                value = xml_entry[tagname]['value']
+            elif isinstance(xml_entry[tagname], basestring):
+                value = xml_entry[tagname]
+
+        if out_prefix_name is not None and not out_prefix_name.isspace():
+            tagname = out_prefix_name + name
+
+        if value is not None:
+            if isinstance(value, in_type):
+                entry[tagname] = value
+            elif in_type == int or in_type == float:
+                number = convert_to_number(value)
+                if number is not None:
+                    entry[tagname] = number
+            elif in_type == datetime:
+                date = convert_to_naive_utc(value)
+                if date is not None:
+                    entry[tagname] = date
+            elif in_type == bool:
+                boolvalue = None
+                if isinstance(value, basestring):
+                    if str_to_boolean(value):
+                        boolvalue = True
+                if not boolvalue:
+                    number = convert_to_number(value)
+                    if number is not None and number > 0:
+                        boolvalue = True
+                    else:
+                        boolvalue = False
+                entry[tagname] = boolvalue
+            else:
+                log.warning('Unsupported attribute type: %s via name: %s' % (in_type, tagname))
+
+    def fill_namespace_attributes(self, xml_entry, entry):
+        for key in NAMESPACE_ATTRIBUTE_MAP:
+            self.set_ns_attribute(key, xml_entry, entry, NAMESPACE_NAME, NAMESPACE_PREFIX, NAMESPACE_ATTRIBUTE_MAP[key])
+
+
+@event('plugin.register')
+def register_plugin():
+    plugin.register(AnidbFeed, 'anidb_feed', api_ver=2, groups=['list'])
```

Comment:
I was assuming this is needed to register the plugin as some sort of input plugin? Seems i looked at the wrong examples, since all i checked also had this 'list' type, without using the list interface. Guess those are all old plugins than.